src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,23 @@
+package duke.task;
+
+public class Deadline extends Task {
+    protected String by;
```

Comment:
You'd probably want to use a more descriptive variable name since `by` can be understood as the user who created the task or the actual deadline of the task